open-craft/ansible-playbooks
# SE-2046 Allows adding a custom (external) nginx proxy service

Adds the `custom-nginx-proxy` playbook, and updates the `nginx-proxy` role to allow external hosts to be proxied, if configured.

Since this change was added as part of SE-2046, it also adds these changes which were required for redeploying an elasticsearch server:
* Adds "force" install to the elasticsearch apt, to allow downgrading the install to 1.15.

**Testing instructions**

See https://github.com/open-craft/ansible-secrets/pull/168 to verify the custom nginx proxy.

**Reviewer**
